### ISSUE TITLE
[BUG] AttributeError: 'GtfsMetadata' object has no attribute 'download_date'

### ISSUE THREAD

**Issue by maximearmstrong** [MobilityData/mobility-database-interface#195]
[Error report](https://console.cloud.google.com/errors/CKLBp_zihsGRcA?time=P7D&refresh=auto&project=top-glass-307718) 

```AttributeError: 'GtfsMetadata' object has no attribute 'download_date'
at create_dataset_entity_for_gtfs_metadata (/workspace/usecase/create_dataset_entity_for_gtfs_metadata.py:235)
at add_dataset_to_source (/workspace/helpers_cf.py:161)
at add_dataset_to_source_cf (/workspace/main.py:128)
at view_func (/layers/google.python.pip/pip/lib/python3.9/site-packages/functions_framework/__init__.py:149)
at dispatch_request (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/app.py:1936)
at full_dispatch_request (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/app.py:1950)
at reraise (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/_compat.py:39)
at handle_user_exception (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/app.py:1821)
at full_dispatch_request (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/app.py:1952)
at wsgi_app (/layers/google.python.pip/pip/lib/python3.9/site-packages/flask/app.py:2447)
```

**Comment by maximearmstrong** [comment]
Fixed in branch 'issue/146/fix-bugs-in-branch'

**Comment by maximearmstrong** [comment]
Already fixed, but in branch `issue/146/add-new-source`. Merged in Staging in #229 and Master in #230.
